src/app/components/blog/components/new-post-form/new-post-form.component.ts

```diff
@@ -7,9 +15,78 @@ import { Component, OnInit } from '@angular/core';
 })
 export class NewPostFormComponent implements OnInit {
 
-  constructor() { }
+  public content: Map<string, string> = new Map();
+
+  public currentPost!: number;
+
+  @Input()
+  public htmlContent: string = 'Enter your text here!'; 
+
+  public config: AngularEditorConfig = {
+    editable: true,
+    spellcheck: true,
+    height: 'auto',
+    minHeight: '5rem',
+    maxHeight: 'auto',
+    width: 'auto',
+    minWidth: '0',
+    translate: 'yes',
+    enableToolbar: true,
+    showToolbar: true,
+    placeholder: 'Enter text here...',
+    defaultParagraphSeparator: 'p',
+    defaultFontName: 'Calibri',
+    toolbarHiddenButtons: [
+      [],
+      ['insertImage', 'customClasses', 'insertVideo', 'toggleEditorMode']
+    ]
+  };
+
+  public postForm!: FormGroup;
+
+  constructor(private formBuilder: FormBuilderService,
+    private formFieldService: FormFieldsService,
+    private fascadeService: FacsadeService) { }
 
   ngOnInit(): void {
+    this.postForm = this.formBuilder.buildPostFormGroup();
+    this.processConstants();
+
+    this.postForm.valueChanges.pipe(
```

Comment:
debounced save feature! I have questions in the meeting on this

Reply:
So cool!